portfolio.js

```diff
@@ -0,0 +1,15 @@
+const openMenu = document.getElementById('menu');
+const closeMenu = document.getElementById('close_menu');
+const navItems = document.querySelector('.desktop-nav');
+
+openMenu.addEventListener('click', () => {
+  navItems.style.display = 'flex';
+});
+
+closeMenu.addEventListener('click', () => {
+  navItems.style.display = 'none';
+});
+
+navItems.addEventListener('click', () => {
+  navItems.style.display = 'none';
+});
```

Comment:
- [ ] You did great so far by adding the ability to close the mobile menu when a user clicks on a menu item, :+1: Byt looking at your project currently, when a user clicks on a menu item on desktop screens, the menu list also disappears and doesn't show uncles the user refreshes the page, this is not Ideal ad not good for user experience. On desktop screens the user should be able to see the menu at all times and click on any item to navigate to that section. I'll suggest you look into this.  :+1: :smile: 

- - Video Reference:
  

https://github.com/Bupilipili/MY-PORTFOLIO/assets/107632780/14d6b1a9-99c1-4584-85b0-4758ba7d40b8